wgkey/key.go

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package wgkey contains types and helpers for WireGuard keys.
```

Comment:
explain the relation to `tailscale.com/types/key`? (Also, why isn't this a sibling under the `types` directory?)

Reply:
> explain the relation to tailscale.com/types/key?

Done, as best I could. It would be nice to unify them someday, but their behavior is different enough that that should happen in follow-up work.

> Also, why isn't this a sibling under the types directory?

Now it is.